### PR TITLE
Scroll Link

### LINKED ISSUE #11: Links take you to correct section
Do we need a header?

### PR DESCRIPTION
**Pull request by Ashepherd214**
Added react-scroll to navigate to sections in page.

### CODE REVIEW

File: src/components/scroll-links.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+import { Element } from 'react-scroll'
+
+export default function () {
+  return (
+    <React.Fragment>
+
+      <Element id='home' name='home'>
+        // wrap your content in the Element from react-scroll
```
Comment (jjarrett21) [inline src/components/scroll-links.js]: This should technically take children as a prop and then return children in the element, can you add a link to where you found this implementation?

**Review by jjarrett21 — CHANGES_REQUESTED** [review]

File: src/components/scroll-links.js (hunk 1 above)
Comment (Ashepherd214) [reply src/components/scroll-links.js]: This file specifically is going to be deleted. I was trying out this implementation directly from a blog I was reading but didn't have a chance to determine for sure whether I needed it or not. I just deleted the file on my local copy and it is not needed so I'll remove that from the request. 

The Element component is the only thing that has been used in each individual section.

**Review by jjarrett21 — APPROVED** [review]

File: src/components/header.js [hunk 2]
```diff
@@ -3,35 +3,94 @@ import { jsx, css } from "@emotion/core"
...
+import { Link, animateScroll as scroll } from "react-scroll"
+import Logo from "../images/sl-logo-short.png"
 
-const wrapperStyle = css`
-  position: relative;
-  display: flex;
-  justify-content: flex-end;
-`
+
+// const wrapperStyle = css`
```
Comment (jjarrett21) [inline src/components/header.js]: remove commented out code

File: src/components/header.js [hunk 3]
```diff
@@ -3,35 +3,94 @@ import { jsx, css } from "@emotion/core"
...
 `
 
+const scrollToTop = function() {
+  scroll.scrollToTop()
+}
+
 const Header = () => {
   return (
-    <div css={wrapperStyle}>
+    // <div css={wrapperStyle}>
```
Comment (jjarrett21) [inline src/components/header.js]: same here

File: src/components/header.js [hunk 4]
```diff
@@ -3,35 +3,94 @@ import { jsx, css } from "@emotion/core"
...
+              smooth={true}
+              duration={500}
+            >
+              Contact
+            </Link>
+          </Nav.Link>
         </Nav>
       </Navbar>
-    </div>
+    // </div>
```
Comment (jjarrett21) [inline src/components/header.js]: here as well

**Review by jjarrett21 — CHANGES_REQUESTED** [review]

**Review by Ashepherd214 — COMMENTED** [review]
Removed the unneeded comments.

**Review by jjarrett21 — APPROVED** [review]

**Pull request merged**